### PR TITLE
feat: add skynet logging

### PR DESCRIPTION
**Pull request by rajeshad**
Add skynet logging

### CODE REVIEW

File: .github/workflows/reusable-build-test-release.yml [hunk 1]
```diff
@@ -1716,3 +1719,16 @@ jobs:
...
+    if: always()
+    needs: [publish]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send logs to Skynet
+        uses: splunk/collect-ta-logs@main
+        with:
+          git_token: ${{secrets.GH_TOKEN}}
+          skynet-token: ${{ secrets.SKYNET_TOKEN }}
+          skynet-url: "https://http-inputs-services-ingest.splunkcloud.com/services/collector/event"
```
Comment (artemrys) [inline .github/workflows/reusable-build-test-release.yml:1733]: Can we make this as a secret as well? So we can change it easier through the https://github.com/splunk/addonfactory-sync-secrets.

**Review by artemrys — COMMENTED** [review]
Do you want it to rolled out automatically? Or we can introduce it as a feature (feat)?

File: .github/workflows/reusable-build-test-release.yml (hunk 1 above)
Comment (rajeshad) [reply .github/workflows/reusable-build-test-release.yml:1733]: You mean move the URL to secrets?

Comment (artemrys) [reply .github/workflows/reusable-build-test-release.yml:1733]: Yes.

Comment (rajeshad) [reply .github/workflows/reusable-build-test-release.yml:1733]: Is URL treated as secret? Are we storing other URLs to the secrets as well?

Comment (rajeshad) [reply .github/workflows/reusable-build-test-release.yml:1733]: @artemrys ?

Comment (artemrys) [reply .github/workflows/reusable-build-test-release.yml:1733]: Sorry, I think I am fine having it here as an open string.

**Review by artemrys — APPROVED** [review]

**Pull request merged**